### PR TITLE
Update risk scoring documentation and screenshot captures to reflect …

### PR DESCRIPTION
**Pull request by quachlingA**
### What does this PR do? What is the motivation?
This PR changes the content of the risk insights documentation with updated sections and added relevant screenshots.

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 1]
```diff
@@ -1,62 +1,58 @@
...
 
 ## Prerequisites
 
-- For AWS entities coverage, [AWS must be configured for Cloud SIEM][1].
-- To view associated Cloud Security Management (CSM) data in the entity panel, [CSM must be configured][2].
+- For AWS Risk Insights coverage, [AWS must be configured for Cloud SIEM][1].
+- To view associated Cloud Security Management (CSM) insights in the entity panel, [CSM must be configured][2].
 
-No other setup is required to use Cloud SIEM Entities.
+Cloud Security Management (CSM) is not required in order to use AWS Risk Insights. No other setup is required to use AWS Risk Insights.
```
Comment (drichards-87) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:25]:
```suggestion
```

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 2]
```diff
@@ -1,62 +1,58 @@
...
 - View all data relevant to an entity, such as signals, misconfigurations, and identity risks.
 - Triage relevant items in bulk.
 - Take mitigation steps such as creating a global suppression or creating a case for an entity.
 
 ## Prerequisites
 
-- For AWS entities coverage, [AWS must be configured for Cloud SIEM][1].
-- To view associated Cloud Security Management (CSM) data in the entity panel, [CSM must be configured][2].
+- For AWS Risk Insights coverage, [AWS must be configured for Cloud SIEM][1].
+- To view associated Cloud Security Management (CSM) insights in the entity panel, [CSM must be configured][2].
```
Comment (drichards-87) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:23]:
```suggestion
- (Optional) To view associated Cloud Security Management (CSM) insights in the entity panel, [CSM must be configured][2].
```

Comment (drichards-87) [reply content/en/security/cloud_siem/entities_and_risk_scoring.md:23]: I recommend labeling CSM as being optional. That'll allow us to delete the part that comes after this, as it's just repeating the same information (that CSM is not required).

**Review by drichards-87 — CHANGES_REQUESTED** [review]

File: content/en/security/cloud_siem/entities_and_risk_scoring.md (hunk 1 above)
Comment (quachlingA) [reply content/en/security/cloud_siem/entities_and_risk_scoring.md:25]: I've gotten the question of whether CSM is required or not in order to use the feature a few times now. Is this something we can add somewhere if not here?

**Pull request merged**